rakefile.rb

```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)

```

Comment:
Interesting. But that's also way different to https://github.com/FakeItEasy/FakeItEasy/graphs/contributors as well (see Patrik), so I don't really know where the latter gets its numbers...